service/history/circuitbreakerpool/outbound_queue.go

```diff
@@ -0,0 +1,44 @@
+// The MIT License
```

Comment:
I had to add to a submodule of `service/history` to go around circular dependency.